psalm-baseline.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="5.12.0@f90118cdeacd0088e7215e64c0c99ceca819e176">
+  <file src="tests/Doctrine/Deprecations/DeprecationTest.php">
+    <DeprecatedMethod>
+      <code>parent::expectDeprecation()</code>
+      <code>parent::expectDeprecationMessage($message)</code>
+    </DeprecatedMethod>
```

Comment:
Although I hinted at a baseline in #46, we usually put this directly and manually in `psalm.xml`, along with a comment. Here we would add a comment explaining that this can be dropped when dropping version X of PHPUnit.

Reply:
Updated.

The version of PHPUnit that triggers them is actually 9.6. It looks like PHPUnit 10 does not allow expecting deprecations anymore: https://github.com/sebastianbergmann/phpunit/issues/5062